app/helpers/chat_channels_helper.rb

```diff
@@ -0,0 +1,48 @@
+# rubocop:disable Rails/HelperInstanceVariable
```

Comment:
Blocking: We should absolutely not ignore this warning. Instance variables have no place in helpers since they indicate that this is not a generic helper, but something that relies on a specific context.